float8_experimental/__init__.py

```diff
@@ -7,4 +7,9 @@
 from float8_experimental.float8_linear import Float8Linear
 from float8_experimental.float8_tensor import Float8Tensor
 
+# Needed to load Float8Tensor with weights_only = True
+from torch.serialization import add_safe_globals
```

Comment:
Spoke with @mikaylagawarecki  and named_tuple support can be added to weight_only

Reply:
https://github.com/pytorch/pytorch/pull/129251